src/app.css

```diff
@@ -0,0 +1,22 @@
+@import url("styles/colors.css");
```

Comment:
you're using 4 space indentation and 2 space indentation. should be 2space across the board